analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -329,3 +335,18 @@ private class PackageManagerRunner(
         }
     }
 }
+
+private fun getResolvedConfiguration(
```

Comment:
Nit: I would prefer `resolveConfiguration` because `getResolvedConfiguration` sounds more like it's getting a previously resolved configuration while `resolveConfiguration` sounds more like it's actually doing the resolution.